src/js/__tests__/components/input/FormiflyMultiSelect.test.tsx

```diff
@@ -5,21 +5,26 @@ import ObjectValidator from '../../../classes/ObjectValidator';
 import StringValidator from '../../../classes/StringValidator';
 import AutomagicFormiflyField from '../../../components/input/AutomagicFormiflyField';
 import FormiflyForm from '../../../components/meta/FormiflyForm';
+import {css} from 'styled-components';
+import 'jest-styled-components';
+
 
 describe.each([
     [true, 'background-color 0s ease-in', 'reduces motion when reduceMotion is true'],
     [false, 'background-color 0.25s ease-in', 'does not reduce motion when reduceMotion is false'],
 ])('Test FormiflyMultiSelect', (reducedMotion, expectedStyle, name) => {
-    test(name, () => {
+    test(name, async () => {
         const shape = new ObjectValidator({foo: new ArrayValidator(new StringValidator())});
-        render(<FormiflyForm shape={shape} onSubmit={() => {}} theme={{reducedMotion: reducedMotion}}>
+        const wrapper = render(<FormiflyForm shape={shape} onSubmit={() => {}} theme={{reducedMotion: reducedMotion}}>
             <AutomagicFormiflyField label="test" name="foo" options={[{value: 'bla', label: 'blub'}]}/>
         </FormiflyForm>);
+        await wrapper.findByText('blub');
 
-        const options = document.getElementsByClassName('formifly-multi-select-option');
-        const style = window.getComputedStyle(options[0]);
+        const options = document.getElementsByClassName('formifly-multi-select-container');
 
-        expect(style.transition).toStrictEqual(expectedStyle);
+        expect(options[0]).toHaveStyleRule('transition', expectedStyle, {
+            modifier: css`menu li`,
+        });
```

Comment:
To be honest, I'm not sure why this test worked before the version bump. The `render` method shouldn't render out css class definitions by default, as far as I understand. I added `jest-styled-components`, because that package does just that - rendering the css classes into the document `render` creates, so we can check for the transition settings.